apps/admin-gui-e2e/src/integration/vo-management-perun_admin.spec.js

```diff
@@ -12,26 +12,268 @@ context('Actions', () => {
+
+    cy.intercept('**/registrarManager/updateFormItems/**').as('addFormItem')
+      .get('[data-cy=test-e2e-vo-from-db]')
+        .click()
+      .get('[data-cy=advanced-settings]')
+        .click()
+      .get('[data-cy=application-form]')
+        .click()
+      .get('[data-cy=add-form-item]')
+        .click()
+      .get('[data-cy=item-short-name]')
+        .type('Header')
+      .get('[data-cy=add-form-item-button-dialog]')
+        .click()
+      .get('[data-cy=edit-form-item-button-dialog]')
+        .click()
+      .get('[data-cy=save-application-form]')
+        .click()
+      .wait('@addFormItem')
+      .get('[data-cy=refresh-button]')
+        .click()
+      // assert that form item exists
+      .get('[data-cy=header-delete]')
+        .should('exist')
+  });
+
+  it('test delete vo application form item', () => {
+
+    cy.intercept('**/registrarManager/updateFormItems/**').as('deleteFormItem')
+      .get('[data-cy=test-e2e-vo-from-db]')
+        .click()
+      .get('[data-cy=advanced-settings]')
+        .click()
+      .get('[data-cy=application-form]')
+        .click()
+      .get('[data-cy=input-test-delete]')
+        .click()
+      .get('[data-cy=delete-application-form-item-dialog]')
+        .click()
+      .get('[data-cy=save-application-form]')
+        .click()
+      .wait('@deleteFormItem')
+      .get('[data-cy=refresh-button]')
+        .click()
+      // assert that form item doesn't exist
+      .get('[data-cy=input-test-delete]')
+        .should('not.exist')
+  });
+
+  it('test add vo manager', () => {
+
+    cy.intercept('**/authzResolver/setRole/**').as('setRole')
+      .get('[data-cy=test-e2e-vo-from-db]')
+        .click()
+      .get('[data-cy=advanced-settings]')
+        .click()
+      .get('[data-cy=managers]')
+        .click()
+      .get('[data-cy=add-manager-button]')
+        .click()
+      .get('[data-cy=search-manager-input]')
+        .type('test')
+      .get('[data-cy=search-manager-button]')
+        .click()
+      .get('[data-cy=test-checkbox]')
+        .click()
+      .get('[data-cy=add-manager-button-dialog]')
+        .click()
+      .wait('@setRole')
+      // assert that manager was added
+      .get('[data-cy=test-checkbox]')
+        .should('exist')
+  });
+
+  it('test remove vo manager', () => {
+
+    cy.intercept('**/authzResolver/unsetRole/**').as('unsetRole')
+      .get('[data-cy=test-e2e-vo-from-db]')
+        .click()
+      .get('[data-cy=advanced-settings]')
+        .click()
+      .get('[data-cy=managers]')
+        .click()
+      .get('[data-cy=vo-manager-checkbox]')
+        .click()
+      .get('[data-cy=remove-manager-button]')
+        .click()
+      .get('[data-cy=remove-manager-button-dialog]')
+        .click()
+      .wait('@unsetRole')
+      // assert that manager doesn't exist
+      .get('[data-cy=vo-manager-checkbox]')
+        .should('not.exist')
+  });
+
+  it('test delete vo', () => {
```

Comment:
This test failed me once because it didn't manage to find the `auto-focused-filter` element. However, I didn't manage to reproduce it. (Therefore, I think it might be a problem with my setup)